app/ui/pages/02_Queries.py

```diff
@@ -13,6 +14,7 @@
 options = {
     "Query Activity": reports_query_activity.report,
     "Top Spenders": reports_top_spenders.report,
+    "Repeated Queries": reports_query_hash.report,
```

Comment:
Do we need to check the behavior change before adding it here?

Reply:
ahhh, yeah. prob should do